db/models.py

```diff
@@ -1 +1,39 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+    description = models.TextField(blank=True)
```

Comment:
I think you should add choices here, since the player can only choose 1 of 4 races